contrib/opencensus-ext-azure/opencensus/ext/azure/common/utils.py

```diff
@@ -58,12 +58,9 @@ def timestamp_to_iso_str(timestamp):
     return to_iso_str(datetime.datetime.utcfromtimestamp(timestamp))
 
 
-# Validate UUID format
-# Specs taken from https://tools.ietf.org/html/rfc4122
+# Validate GUID format
 uuid_regex_pattern = re.compile('^[0-9a-f]{8}-'
```

Comment:
Is this just a vanilla UUID now? If yes, could validate with [uuid.UUID](https://docs.python.org/3/library/uuid.html#uuid.UUID). Otherwise, the regex looks reasonable to me but im not too confident

Reply:
This is the same regex used in other internal Microsoft SDKs, so I would be consistent.